### ISSUE TITLE
one-space-between-tags does not work at examples level.

### ISSUE THREAD

**Issue by xgbuils** [vsiakka/gherkin-lint#168]
The following gherkin code:

``` gherkin
Feature: Feature with wrong tag spacing

Scenario Outline: This is a Scenario Outline with wrong tag spacing
  Then this is a then step <foo>
@tag1      @tag2
Examples:
  | foo |
  | bar |
```
does't report an error.
